docs/source/configuration.rst

```diff
@@ -522,7 +522,7 @@ You already know you can pass arguments (:code:`--verbose`,
 :code:`--exclude-rules`, etc.) through the CLI commands (:code:`lint`,
 :code:`fix`, etc.):
 
-.. code-block:: console
+.. code-block:: text
```

Comment:
Does this one make any difference?

Reply:
kind of becuase a `$` at the start isn't valid shell code and also it's consistent with every other code block so think we should leave as `text`?

Reply:
Fair enough.